esp/mlrs-wireless-bridge/mlrs-wireless-bridge.ino

```diff
@@ -102,25 +102,30 @@ Troubleshooting:
 //**********************//
 //*** WiFi settings ***//
 
+#define USE_BROADCAST_FOR_UDP // Comment out if you want to use only the first WiFi client to connect
+
 // for TCP, UDP (only for these two)
 // ssid = "" results in a default name, like "mLRS-13427 AP UDP"
 // password = "" makes it an open AP
 String ssid = ""; // "mLRS AP"; // Wifi name
 String password = ""; // "thisisgreat"; // WiFi password
 
+// Our IP address in AP mode
 IPAddress ip(192, 168, 4, 55); // connect to this IP // MissionPlanner default is 127.0.0.1, so enter 192.168.4.55 in MP
 
 int port_tcp = 5760; // connect to this port per TCP // MissionPlanner default is 5760
 int port_udp = 14550; // connect to this port per UDP // MissionPlanner default is 14550
 
-// for UDPCl (only for UDPCl)
+// for UDPCl (only for UDPCl) Station mode
 // network_ssid = "" results in
 // - a default name, like "mLRS-13427 STA UDPCl"
 // - and a default password which includes your mLRS bindphrase, like "mLRS-mlrs.0"
 String network_ssid = ""; // name of your WiFi network
 String network_password = "****"; // password to access your WiFi network
 
-IPAddress ip_udpcl(192, 168, 0, 164); // connect to this IP // MissionPlanner default is 192.168.0.164
+// Uncomment and edit the next two lines only if your AP has no DHCP server
+//#define FORCED_CLIENT_ADDRESS
+//IPAddress ip_udpcl(192, 168, 0, 164); // Our static IP in STA mode // Configure MissionPlanner UDPCl with this address
```

Comment:
this doesn't have to be outcommented, right? (or does it give a compile warning?)
I don't like the change of the comment ... STA is somewhat superfluous, "Configure" sounds scary, and I miss that it's MP's default

Reply:
But, this is not MP's default!  MP's default is 128.0.0.1 (localhost).  This is some random address that you said you forgot where it came from.  Our UDPCl mode uses WiFi Station mode and we should get our address via DHCP and should NEVER need to use a statically configured address like this.  IE, this should never have been here in the overwhelming common case.

Reply:
I don't want to leave line 128 uncommented because that might lead the user to believe that this address is being used for something when, in fact, it's just some random IP address which serves no purpose in the normal case.

Reply:
> But, this is not MP's default! MP's default is 128.0.0.1 (localhost).

in fact, that's correct ... checked on a pristine system. That's confusing now. Anyway.